### PR TITLE
Monitor Mithril nodes footprint

### PR DESCRIPTION
**Pull request by jpraynaud**
## Content
This PR includes improvements to the monitoring of the Mithril nodes to benchmark their full footprint:
- Monitoring of the Mithril nodes resources (CPU, memory)
- Monitoring of the Mithril aggregator and signer disk usage

## Pre-submit checklist

- Branch
  - [x] Crates versions are updated (if relevant)
  - [x] CHANGELOG file is updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested

## Issue(s)
Relates to #1826

### CODE REVIEW

**Review by sfauvel — APPROVED** [review]
LGTM

File: mithril-infra/assets/docker/prometheus/disk_usage_exporter/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Precreate workdir
+RUN mkdir -p /app/bin
+
+# Upgrade
+RUN apt-get update -y && apt-get install -y libssl-dev ca-certificates wget && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+#Workdir
+WORKDIR /app/
+
+## Donwload Disk usage exporter
```
Comment (dlachaume) [inline mithril-infra/assets/docker/prometheus/disk_usage_exporter/Dockerfile:16]:
```suggestion
## Download Disk usage exporter
```

File: mithril-infra/assets/docker/docker-compose-monitoring.yaml [hunk 2]
```diff
@@ -52,6 +52,44 @@ services:
...
         tag: "{{.ImageName}}|{{.Name}}|{{.ImageFullID}}|{{.FullID}}"
 
+  process-exporter:
+    image: ncabatoff/process-exporter:latest
+    container_name: process-exporter
+    profiles:
+      - tools
+      - monitoring
+      - all
+    restart: unless-stopped
```
Comment (dlachaume) [inline mithril-infra/assets/docker/docker-compose-monitoring.yaml:62]: Why aren't the two processes configured the same way?

```
process-exporter:
  restart: unless-stopped
```

```
disk-usage-exporter:
    restart: always
```

File: mithril-infra/assets/docker/prometheus/disk_usage_exporter/Dockerfile (hunk 1 above)
Comment (jpraynaud) [reply mithril-infra/assets/docker/prometheus/disk_usage_exporter/Dockerfile:16]: Done

File: mithril-infra/assets/docker/docker-compose-monitoring.yaml (hunk 2 above)
Comment (jpraynaud) [reply mithril-infra/assets/docker/docker-compose-monitoring.yaml:62]: I switched it to `always`

**Pull request merged**